tools/serverpod_cli/test/test_util/builders/enum_definition_builder.dart

```diff
@@ -74,6 +77,12 @@ class EnumDefinitionBuilder {
     return this;
   }
 
+  EnumDefinitionBuilder withDefaultValue(
```

Comment:
@SandPod I still kept this, but now it doesn't have tests using it because you told me to move it to the end 2 end tests. What should I do with this?